lib/main_common.pm

```diff
@@ -510,7 +510,7 @@ sub installzdupstep_is_applicable {
 
 sub snapper_is_applicable {
     my $fs = get_var("FILESYSTEM", 'btrfs');
-    return ($fs eq "btrfs" && get_var("HDDSIZEGB", 10) > 10);
+    return ($fs eq "btrfs" && !get_var("LIVETEST") && get_var("HDDSIZEGB", 10) > 10);
```

Comment:
It looks weird to put the LIVETEST between two filesystem related settings. Please move to the beginning or the end. Then, LGTM